dwhdelft.nl/components/pages/mixup/LinkedEvents.vue

```diff
@@ -47,22 +47,38 @@ onMounted(() => {
 
 <template>
   <section
-    v-if="events && events.length > 0"
+    v-if="events === null || events.length > 0"
```

Comment:
dit moet `events !== null` zijn lijkt me?

Reply:
Als `events` gelijk is aan `null`, dan is er nog niks geset, dus zijn we nog aan het loaden.

Als `events` een lege array is hebben we wel wat geset, en dan zijn we klaar met loaden.

Vandaar dat we ook de case willen dat het `null` is en onderstaand dus tonen in alle gevallen behalve `events.length === 0`

Reply:
Ah top! Dank voor de uitleg